Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6-onbuild
+
+RUN ./script/generatecert
+EXPOSE 8080
+
+CMD app \
+  -addr=localhost:8080 \
```

Comment:
The issue may be that you're binding to the `localhost` loopback interface. Try `-addr=0.0.0.0:8080` instead